reference/attributes.rst

```diff
@@ -123,6 +123,8 @@ Twig
 ~~~~
 
 * :ref:`Template <templates-template-attribute>`
+* :ref:`AsTwigFilter <templates-twig-filter-attribute>`
+* :ref:`AsTwigFunction <templates-twig-function-attribute>`
```

Comment:
What about also adding something for the `#[AsTwigTest]` attribute?

Reply:
I added `#[AsTwigTest]` to the list, but just as text, without any link to it. I think this is rare enough to not explain it and just mention that it exists. Thanks.